scrapenews/spiders/sabc.py

```diff
@@ -46,8 +47,10 @@ def parse_item(self, response):
             article_body = response.css('div.post-content')
             body_html = " ".join(article_body.css('::text').extract())
             byline = response.css('span.author::text').extract_first().strip()
-            publication_date_str = response.css('span.create::text').extract_first().strip()
```

Comment:
why are we changing from a css selector to xpath? css is to be preferred because xpath breaks when there's a second class - it doesn't understand the semantics of the class attribute (any order, separated by spaces). As far as I can tell, your xpath is precisely the same as what the css used to do, but the css was more robust.